bootstrap/kubeadm/internal/cloudinit/controlplane_join.go

```diff
@@ -54,10 +179,21 @@ func NewJoinControlPlane(input *ControlPlaneJoinInput) ([]byte, error) {
 	// TODO: Consider validating that the correct certificates exist. It is different for external/stacked etcd
 	input.WriteFiles = input.Certificates.AsFiles()
 	input.WriteFiles = append(input.WriteFiles, input.AdditionalFiles...)
+	joinScript, err := generate("JoinControlplaneScript", controlPlaneJoinScript, input)
```

Comment:
if RetriableBootstrap feature gate enabled?

Reply:
noted, i just haven't wired that up yet.